AdminWebsite/AdminWebsite/Controllers/HearingsController.cs

```diff
@@ -43,6 +43,7 @@ public class HearingsController : ControllerBase
         private readonly IUserAccountService _userAccountService;
         private readonly IUserIdentity _userIdentity;
         private readonly IPublicHolidayRetriever _publicHolidayRetriever;
+        private static readonly int thirty = 30;
```

Comment:
Call this something more descriptive.

startingSoonMinutesThreshold maybe, just in case they ever want to change it from 30 to say 60 or something.